python/cuml/dask/common/dask_arr_utils.py

```diff
@@ -163,3 +164,28 @@ def _conv_np_to_df(x):
 
         return dask.array.from_delayed(final_result, shape=shape,
                                        meta=meta)
+
```

Comment:
Is this code copied from my `make_blobs` PR or was it pulled from there? The diff seems to be misleading. If copied, it is good practice to preserve the version history

Reply:
Good point. I fixed it to keep preserve the version history.